client/actions/locations.js

```diff
@@ -1,5 +1,112 @@
-import {formatAddress} from '../utils';
+import {formatAddress, getErrorMessage} from '../utils';
 import {get} from 'lodash';
+import {MODALS} from '../constants';
+import {showModal} from './index';
+
+
+const locationsSearch = (searchQuery) => (dispatch) => {
+    dispatch(
+        {
+            type: 'LOCATIONS_CHANGE_LIST_SETTINGS',
```

Comment:
Create a constants file for location actions (and use these constants in both actions and reducers).

Reply:
Done!